voluncheer/profiles/models.py

```diff
@@ -219,3 +219,30 @@ def award_volunteer_hours_badges(self):
             self.save()
 
         return hours_remaining
+
+
+def _post_photo_path(instance, filename):
+    return f"opportunities/{instance.volunteer.user.id}/{filename}"
+
+
+class GallaryPost(models.Model):
+    """The GalaryPost type.
+
+    Attributes:
+        volunteer: post belong to.
+        author: post author, either created by the same volunteer or aliens.
```

Comment:
hahah. But seriously, why do we need need separate fields for the author and the volunteer who created it?

Reply:
volunteer is the post belong to. Say author is vol A and they write a post to vol B, vol A is the author and vol B is the belonged volunteer

Reply:
I see, so the post appears on the profile of `volunteer`, but other users can write posts on that profile.

Reply:
maybe...

   Attributes:
        volunteer: The owner of the post. Posts appear on this volunteer's profile.
        author: The author of the post. A volunteer can post on any other volunteer's profile.

Reply:
Okay, Changed.